ui/app/helpers/utils/common.util.js

```diff
@@ -1,5 +1,5 @@
 export function camelCaseToCapitalize (str = '') {
   return str
     .replace(/([A-Z])/ug, ' $1')
-    .replace(/^./u, (str) => str.toUpperCase())
+    .replace(/^./u, (s) => s.toUpperCase())
```

Comment:
Less worried about this one, but raising cause I raised about the other instances.

Reply:
I left this one but fixed the others (I couldn't think of  a better replacement)